skyfield/timelib.py

```diff
@@ -461,6 +479,17 @@ def utc_datetime_and_leap_second(self):
             dt = datetime(year, month, day, hour, minute, second, milli, utc)
         return dt, leap_second
 
+    def timestamp(self):
```

Comment:
As Python 2 doesn’t support this concept, I think I prefer that we not add it to Skyfield. But I should back up and ask: what are the advantages of datetime timestamp values? What use case would be helped by bringing them into Skyfield? I've not yet seen any astronomy code that uses them.

Reply:
I just noticed that find_discrete() should work fine with Time objects. If that's the case then the entire code could be slimmed down quite a bit

Reply:
A lot of time has passed. I thought I added the timestamp methods to satisfy the parameters of find_discrete(), but I was wrong. I believe I added them for simplicity or because the project I was working on at the time was using timestamps and datetime but I don't remember exactly

Reply:
In that case, let’s definitely go for slimming the code down and making it simpler, and later if folks want to call this routine with timestamps, we can add documentation showing them how to do the conversion themselves. Thanks!